src/web/app.py

```
@@ -1086,6 +1096,43 @@ async def get_feed(
     return result
 
 
+@app.post("/api/feed/briefing")
+async def generate_feed_briefing(
+    tz: str | None = None,
+    current_user: dict = Depends(get_current_user),
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)